### DISCUSSION TITLE
Lenovo legion 2023

### DISCUSSION THREAD

**Discussion by vttiste** [4JX/L5P-Keyboard-RGB#109] [Ideas]
Hi,
any plans to add the new Lenovo Legion from 2023 ? I have tried with Lenovo Legion Pro 5 16IRX8 but so far I get message that the hardware is not supported. This said I believe the keyboard should be the same or very similar to the 2022 version.
Cheers

**Comment by HyperN00B** [comment]
Install LenovoLegionLinux and lenovo-legion-wmi-git for better integration, and run the app with sudo. Running as a regular user will result in the error you mentioned even if the HW itself is supported.
